server/src/main/java/com/objectcomputing/checkins/services/member_skill/MemberSkillCreateDTO.java

```diff
@@ -17,6 +18,12 @@
     @Schema(description = "the id of the skill", required = true)
     private UUID skillid;
 
+    @Schema(description = "the level of the skill")
```

Comment:
For the description here could we say something like "member's expertise level for this skill"?

Reply:
This is done.